Update Redis Connector Example Page (#446)
wso2/docs-mi: en/docs/reference/connectors/redis-connector/redis-connector-example.md

5. To create the other two resources, click on the **+ Resource** button on the **Service Designer** and configure the following values.

      <table>
         <tr>
            <th>Parameter</th>
            <th>resource 2</th>
            <th>resource 3</th>
         </tr>
         <tr>
           <td>Url Style</td>
           <td><code>URL_TEMPLATE</code></td>
           <td><code>URL_TEMPLATE</code></td>
         </tr>
         <tr>
           <td>Uri Template</td>
           <td>
             <code>/getstockvolumedetails</code>
           </td>
           <td>
             <code>/deletestockvolumedetails</code>
           </td>
         </tr>
         <tr>
           <td>Methods</td>
           <td><code>GET</code></td>
           <td><code>POST</code></td>
         </tr>
       </table> 

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/add-resource.png" title="Add Resource" width="600" alt="Add Resource"/>

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/resource-overview.png" title="List of Resources" width="600" alt="List of Resources"/>

### Create the mediation logic

#### Configure a resource for the getstockquote operation

> Configure a resource that sets up Redis hash map and sets a specific field in a hash to a specified value. In this sample, the user sends the request to invoke the created API to get WSO2 stock volume details. To achieve this, add the follow the steps below.

Reviewer: ```suggestion
Configure a resource that sets up Redis hash map and sets a specific field in a hash to a specified value. In this sample, the user sends the request to invoke the created API to get WSO2 stock volume details. To achieve this, follow the steps below.
```

> 1. Click on the `GET getstockquote` API resource under available resources on the Service Designer.

Reviewer: ```suggestion
1. Click on the `GET getstockquote` API resource under available resources on the **Service Designer**.
```

    You will now see the graphical view of the `SampleRedisAPI` with its default API Resource.

> 2. Click on the **+** icon under the **start** to add a mediator.

Reviewer: ```suggestion
2. Click on the **+** icon under **start** to add a mediator.
```

> 3. Add a payload factory mediator from the mediator palette to extract the selected stock details. In this sample, we attempt to get WSO2 stock details from the SimpleStockQuoteService.

Reviewer: ```suggestion
3. Add a payload factory mediator from the mediator palette to extract the selected stock details. In this sample, we attempt to get WSO2 stock details from the `SimpleStockQuoteService`.
```

    <table>
      <tr>
         <th>Parameter</th>
         <th>Value</th>
      </tr>
      <tr>
        <td>Payload Format</td>
        <td><code>Inline</code></td>
      </tr>
      <tr>
        <td>Media Type</td>
        <td>
          <code>xml</code>
        </td>
      </tr>
      <tr>
        <td>Template Type</td>
        <td><code>default</code></td>
      </tr>
      <tr>
        <td>Payload</td>
        <td>
         ```
         <m0:getQuote xmlns:m0="http://services.samples">
         <m0:request>
         <m0:symbol>$1</m0:symbol>
         </m0:request>
         </m0:getQuote>
         ```
        </td>
      </tr>
    </table>

    Add parameter:
    <table>
      <tr>
         <td>Argument value</td>
         <td><code>get-property('uri.var.symbol')</code></td>
      </tr>
    </table>

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/add-payload-factory.png" title="Add payloadfactory to extract WSO2 details" width="600" alt="Add payloadfactory to extract WSO2 details"/> 

> 4. Add a header to get a quote from the SimpleStockQuoteService.

Reviewer: ```suggestion
4. Add a header to get a quote from the `SimpleStockQuoteService`.
```

    <table>
         <tr>
            <th>Parameter</th>
            <th>Value</th>
         </tr>
         <tr>
           <td>Header Name</td>
           <td><code>Action</code></td>
         </tr>
         <tr>
           <td>Header Action</td>
           <td>
             <code>set</code>
           </td>
         </tr>
         <tr>
           <td>Scope</td>
           <td><code>default</code></td>
         </tr>
         <tr>
           <td>Header Value Type</td>
           <td><code>LITERAL</code></td>
         </tr>
         <tr>
           <td>Header Value Literal</td>
           <td><code>urn:getQuote</code></td>
         </tr>
       </table> 

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/redis-header.png" title="Add Header to get Quote" width="600" alt="Add Header to get Quote"/>

> 6. Add an address endpoint using the call mediator to access SimpleStockQuoteService. Select the previously created `StockQuoteEP` from the dropdown.

Reviewer: ```suggestion
6. Add an address endpoint using the call mediator to access `SimpleStockQuoteService`. Select the previously created `StockQuoteEP` from the dropdown.
```

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/call-mediator.png" title="Call mediator" width="600" alt="Call mediator"/>  

8. In this example, we copy the original payload to a property using the Enrich mediator.
   
    <img src="{{base_path}}/assets/img/integrate/connectors/redis/enrich-mediator-1.png" title="Add enrich mediator" width="600" alt="Add enrich mediator"/> 

> 9. To get the input values in to the `hSet`, we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators onto the Design pane as shown below.

Reviewer: ```suggestion
9. To get the input values in to the `hSet`, we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then add the `Property` mediators onto the Design pane as shown below.
```

>    1. Add the property mediator to capture the `symbol` value from the response of SimpleStockQuoteService. The 'symbol' contains the company name of the stock quote.

Reviewer: ```suggestion
   1. Add the property mediator to capture the `symbol` value from the response of `SimpleStockQuoteService`. The 'symbol' contains the company name of the stock quote.
```

    <img src="{{base_path}}/assets/img/integrate/connectors/redis/enrich-mediator-2.png" title="Add enrich mediator" width="500" alt="Add enrich mediator"/> 

13. Forward the backend response to the API caller.
    
     When you are invoking the created resource, the request of the message is going through the `/getstockquote` resource. Finally, it is passed to the [Respond mediator]({{base_path}}/reference/mediators/respond-mediator/). The Respond Mediator stops the processing on the current message and sends the message back to the client as a response.            
    
     1. Add the **respond mediator** to the **Resource view**. 

     2. Once you have setup the resource, you can see the `getstockquote` resource as shown below.
    
          <img src="{{base_path}}/assets/img/integrate/connectors/redis/getStockQuote-resource.png" title="getStockQuote resource" width="400" alt="getStockQuote resource"/>

#### Configure a resource for the getstockvolumedetails operation
    
1. Navigate to the **Service Designer** view of the API and select the `getstockvolumedetails` resource.

> 2. Click the **+** under the **start** and select the **Redis** connector from the **Connectors** section.

Reviewer: ```suggestion
2. Click **+** under **start** and select the **Redis** connector from the **Connectors** section.
```
